### DISCUSSION TITLE
Specifying Account in CSV import

### DISCUSSION THREAD

**Discussion by ups7** [ghostfolio/ghostfolio#2032] [Q&A] [answered]
Is it possible to specify the account in the CSV import? I tried uploading some activities using the template given but it always uploads with a blank account. If I were to upload hundreds of transactions, it will be very painful to update them all one by one.

Thanks in advance!

**Comment by MarcoCaneve** [comment, marked as answer]
Is possibile. Just add the column "Account" in the csv and fill it with the name of the account

**Reply by ups7** [reply]
Thank you!

**Reply by ups7** [reply]
Is there a list of all the columns supported? Just tried with Note, and that didn't work.

**Reply by dtslvr** [reply]
You can find the column mapping [here](https://github.com/ghostfolio/ghostfolio/blob/main/apps/client/src/app/services/import-activities.service.ts#L16).

Comment / Note is not yet supported.

**Reply by ups7** [reply]
Thanks @dtslvr!

**Reply by dtslvr** [reply]
I have created a ticket for the comment / note import: #2048
